functions-methods.js

```diff
@@ -19,8 +26,23 @@
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
+console.log("--------------------- opdracht 2 ---------------------")
 
+function typeOfEmail(email){
+    if ( email.includes("@novi.nl")){
+        return "Medewerker";
+    }
+    if ( email.includes("@novi-education.nl")){
```

Comment:
Ook al zorgt de return er al voor dat je niet op deze regel kan komen, dan zou ik tóch nog een `else if` gebruiken. Staat gewoon een stukje netter in mijn ogen